pkg/remotedialer/client.go

```diff
@@ -2,39 +2,42 @@ package remotedialer
 
 import (
 	"context"
-	"net/http"
-	"time"
-
 	"github.com/gorilla/websocket"
 	"github.com/sirupsen/logrus"
+	"net/http"
+	"strings"
```

Comment:
This should be put along with other golang libraries. Third party libraries and golang libraries should be separated. If you run `goimports` it will format for you.